dart/collision/bullet/BulletCollisionNode.h

```diff
@@ -59,11 +60,11 @@ namespace collision {
 class BulletCollisionNode;
 class BulletCollisionDetector;
 
-struct BulletUserData {
-    dynamics::BodyNode* bodyNode;
-    dynamics::ConstShapePtr shape;
-    BulletCollisionNode* btCollNode;
-    BulletCollisionDetector* btCollDet;
+struct BulletUserData
+{
+  dynamics::ShapeNode* shapeNode;
```

Comment:
Should we maybe use a ``WeakNodePtr`` here, or is this something that's strictly internal and will automatically be destroyed if the ``ShapeNode`` is destroyed?